ompi/mpi/c/comm_get_name.c.in

```diff
@@ -44,7 +45,9 @@ PROTOTYPE ERROR_CLASS comm_get_name(COMM comm, STRING_OUT name, INT_OUT length)
     if ( MPI_PARAM_CHECK ) {
         OMPI_ERR_INIT_FINALIZE(FUNC_NAME);
 
-        if ( ompi_comm_invalid ( comm ) )
+        /* Note that MPI 4.1. explicitely allows to invoke comm_get_name
+           on MPI_COMM_NULL  */
+        if (ompi_comm_invalid(comm) && MPI_COMM_NULL != comm)
```

Comment:
Assumedly the rest of this function will work properly if `MPI_COMM_NULL` is passed (e.g., return the string `MPI_COMM_NULL`)?